### DISCUSSION TITLE
Designable 接入问题 CompositePanel

### DISCUSSION THREAD

**Discussion by GadflyX** [alibaba/formily#2122] [Q&A 2.x]
开启严格模式下 , CompositePanel.item 会报如下错误
JSX 元素类型“CompositePanel.Item”不具有任何构造签名或调用签名
修改 CompositePanel 的类型定义 将其 item 改为 非 optional ,可以通过。但拖拽到面板时无效，什么都不会触发。
大纲树有其节点，却没在 workSpace 上显示。

**Comment by janryWang** [comment]
现在在重构dn，马上就出1.0了，建议先等等

**Reply by febugcoder** [reply]
请问有大概的发版时间吗

**Reply by GadflyX** [reply]
有点着急用，可以大致说下是什么问题么？
